test/application_system_test_case.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
```

Comment:
you don't need this folder at all because we use Rspec for testing. So you can simply remove the `test` folder